decidim-core/app/views/decidim/pages/_tabbed.html.erb

```diff
@@ -21,7 +21,7 @@
       <ul id="dropdown-menu-pages" class="vertical-tabs__list" role="menu">
         <% pages.each do |sibling| %>
           <li class="<%= "is-active" if page == sibling %>" role="menuitem">
-            <%= link_to translated_attribute(sibling.title), page_path(sibling.slug) %>
+            <%= link_to translated_attribute(sibling.title), page_path(sibling.slug), "aria_current": (page == sibling).to_s %>
```

Comment:
```suggestion
            <%= link_to translated_attribute(sibling.title), page_path(sibling.slug), "aria-current": (page == sibling).to_s %>
```

Reply:
Use this button everything should be ok. 
![image](https://github.com/user-attachments/assets/8222f8c2-2993-47fe-86a9-cf2e6b774e2a)